app/views/catalog/_containers.html.erb

```diff
@@ -6,6 +6,6 @@
   <% if document.containers.present? && !online_contents_context? %>
     <%= document.containers.join(', ') %>
   <% elsif document.children? && document.level != 'collection' %>
-    <%= link_to_document document, t('dul_arclight.views.show.containers.see_all') %>
+    <% if false %><%= link_to_document document, t('dul_arclight.views.show.containers.see_all') %><% end %>
```

Comment:
Did you just comment out code with an if false block?  We need more cops! lol

Reply:
Yes, guilty. :meowshame: